package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typescript-plugin-css-modules",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Do you mind reverting this line?

Reply:
I'll do this at release... it might be 2.1.0.

Reply:
reverted.